packages/amplify-cli/src/__tests__/amplify-exception-handler.test.ts

```diff
@@ -78,8 +81,10 @@ describe('test exception handler', () => {
     reportErrorMock.mockRejectedValueOnce(new Error('MockTestError'));
     await handleException(amplifyError);
 
-    expect(printerMock.error).toHaveBeenCalledWith(amplifyError.message);
-    expect(printerMock.info).toHaveBeenCalledWith(amplifyError.details);
+    expect(printerMock.error).toHaveBeenCalledWith(`${amplifyError.message}${EOL}${amplifyError.details}`);
+    expect(printerMock.info).toHaveBeenCalledTimes(2);
+    expect(printerMock.info).toHaveBeenNthCalledWith(1, `Resolution: ${amplifyError.details}`);
```

Comment:
This should be `${amplifyError.resolution}`

I lines 76 to 78, use different strings so we don't hide this error.

Reply:
Good point. I also made the error mock more explicit 63146cf9f7e11071c7a03a85130b83f9c47b68b5